presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/ThriftSessionProperties.java

```diff
@@ -13,25 +13,38 @@
  */
 package com.facebook.presto.connector.thrift;
 
+import com.facebook.presto.spi.ConnectorSession;
 import com.facebook.presto.spi.session.PropertyMetadata;
 import com.google.common.collect.ImmutableList;
 
 import javax.inject.Inject;
 
 import java.util.List;
 
+import static com.facebook.presto.spi.session.PropertyMetadata.booleanProperty;
+
 /**
  * Internal session properties are those defined by the connector itself.
  * These properties control certain aspects of connector's work.
  */
 public final class ThriftSessionProperties
 {
+    private static final String SET_THRIFT_IDENTITY_HEADER = "thrift_connector_set_identity_headers";
```

Comment:
Also, we might as well just call this `USE_THRIFT_IDENTITY_HEADER = "use_thrift_identity_header"`. We don't need "thrift_connector" here because this session property will be prefixed with `<connector name>.`.